src/components/CourseCard/CourseCard.tsx

```diff
@@ -0,0 +1,68 @@
+'use client';
+
+import {
+  Button,
+  Card,
+  CardContent,
+  Collapse,
+  Typography,
+  useTheme,
+} from '@mui/material';
+import { Course } from '@prisma/client';
+import { useState } from 'react';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+
+type Props = {
+  course: Course;
+};
+
+const CourseCard = ({ course }: Props) => {
+  const theme = useTheme();
+  const [expanded, setExpanded] = useState(false);
+
+  return (
+    <Card
+      sx={{
+        background: theme.palette.darkBlue.main,
+        color: theme.palette.white.main,
+        width: 340,
+        maxWidth: '100%',
+        borderRadius: '15px',
+        margin: 'auto',
+        textAlign: 'center',
+      }}
+    >
+      <CardContent sx={{ overflowWrap: 'break-word' }}>
+        <Typography variant="h5" m={2}>
+          {course.name}
+        </Typography>
+        <Typography>{course.startDate.toDateString()}</Typography>
```

Comment:
Perhaps also add an ending date for courses spanning over several days?

```suggestion
        <Typography>
          {course.startDate.toDateString()}
          {course.startDate.toDateString() !== course.endDate.toDateString()
            ? " – " + course.endDate.toDateString()
            : null
          }
        </Typography>
```

Reply:
Yes, this is a great idea